llvm/lib/DWARFLinker/DWARFStreamer.cpp

```diff
@@ -297,7 +297,7 @@ void DwarfStreamer::emitDebugNames(DWARF5AccelTable &Table) {
 
   // Build up data structures needed to emit this section.
   std::vector<std::variant<MCSymbol *, uint64_t>> CompUnits;
-  DenseMap<unsigned, size_t> UniqueIdToCuMap;
+  DenseMap<unsigned, unsigned> UniqueIdToCuMap;
```

Comment:
This is no longer a map just to CUs, is it? So maybe `UniqueIdToUnitMap`?

Reply:
Or maybe not... why is this changing then, I guess?

Reply:
This was to fix a Windows build bot. From what I remember it was complaining about lose of preciscsion or something like that? Been couple of weeks. :)

Also size_t didn't really make sense anyway since Id, below, is unsigned.